app.rb

```diff
@@ -0,0 +1,182 @@
+require_relative './student'
+require_relative './teacher'
+require_relative './rental'
+require_relative './book'
+
+# rubocop:disable Metrics/ClassLength
```

Comment:
- [ ] **[OPTIONAL]** I know that at this stage code can be messy and hard to separate concerns, but it would be great if you can find a way of doing the same without disabling any linter rules, for example, you could extract Menu logic into a separate module and use it here whenever you need it! 🤓 🔧